### PR TITLE
feat(client, formatter): forge fmt

### PR DESCRIPTION
**Pull request by shekhirin**
Feel free to test it on your codebase!

1. Package the extension with `npm run package`
2. Import the extension in VS Code with `Install from VSIX...` from `solidity-0.0.138.vsix` in the `vscode-solidity` project root
   image
3. Choose `forge` as a formatter in settings
   image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -148,12 +150,13 @@
         },
         "solidity.formatter": {
           "type": "string",
-          "default": "prettier",
+          "default": "forge",
```
[the remark this answers is not in the document]
Comment (shekhirin) [reply package.json]: yep, agree! changed back. was mostly needed it for testing 😄

File: package.json [hunk 2]
```diff
@@ -151,9 +153,10 @@
           "default": "prettier",
           "enum": [
             "none",
-            "prettier"
+            "prettier",
+            "forge"
           ],
-          "description": "Enables / disables the solidity formatter (prettier solidity default)"
+          "description": "Enables / disables the solidity formatter (foundry forge default)"
```
Comment (joejordan) [inline package.json:159]: Missed this when changing back to prettier default

**Review by risingsun007 — COMMENTED** [review]
Tested it and it worked.

**Comment by juanfranblanco** [comment]
Thank you!!

**Comment by Valentin-Seehausen** [comment]
Thank you, this is great.

Small thing: The formatter does not recognize formatting options written in the `foundry.toml` file. 

Is there a way to apply those configurations?

**Comment by juanfranblanco** [comment]
@shekhirin ^^

**Comment by shekhirin** [comment]
yep, it's currently not respected, can confirm! Opened a PR to set current working directory during formatting: https://github.com/juanfranblanco/vscode-solidity/pull/363

**Pull request merged**